src/lib.rs

```diff
@@ -75,21 +87,19 @@ pub fn get_version() -> Result<NEARLedgerAppVersion, NEARLedgerError> {
     match transport.exchange(&command) {
         Ok(response) => {
             log::info!(
-                "APDU out: {}\nAPDU ret code: {:x}, {:?}",
+                "APDU out: {}\nAPDU ret code: {:x}",
                 hex::encode(response.apdu_data()),
                 response.retcode(),
-                response.error_code()
             );
             // Ok means we successfully exchanged with the Ledger
             // but doesn't mean our request succeeded
             // we need to check it based on `response.retcode`
             if response.retcode() == RETURN_CODE_OK {
                 return Ok(response.data().to_vec());
             } else {
-                let error_string = response
-                    .error_code()
```

Comment:
`ledger-apdu-0.10.0` error code conversion table is probably from another app, its codes do not correspond to the ones in the `app-near` ledger repo